packages/components/grid/src/column.ts

```diff
@@ -187,7 +187,7 @@ export class GridColumn<T = any> extends LitElement {
         </td>
       `;
     } else {
-      return html`<td class=${classes.join(' ')} part=${parts.join(' ')}>${data || 'No path set'}</td>`;
+      return html`<td class=${classes.join(' ')} part=${parts.join(' ')}>${data ?? 'No path set'}</td>`;
```

Comment:
Maybe we should use translation here?

Reply:
I don't think so; this is more of a developer message. Not meant to be seen by end-users.